README.md

```diff
@@ -2,23 +2,30 @@
 
 This repository contains [Dockerfiles](https://docs.docker.com/reference/builder/) for use with [Docker](https://www.docker.com/) and [Compose](https://docs.docker.com/compose/) to deploy [Project Clearwater](http://www.projectclearwater.org).
 
+## Preparation
```

Comment:
It feels a bit odd having multiple sections named "Preparation", and I could see this causing confusion.  Maybe we should call this "Common preparation", and add an introductory sentence above saying that there are two options for installation, and that you need to go through common preparation and then either the compose or non-compose install processes.